src/tv2_afvd_showstyle/config-manifests.ts

```diff
@@ -262,6 +269,57 @@ export const schemaConfigManifest: ConfigManifestEntry[] = [
 	}
 ]
 
+// @todo: use these with SOF-1092:
+const DESIGN_TABLE_ID = 'GFXDesignTemplates'
+const DESIGN_NAME_COLUMN_ID = 'INewsName'
+
+export const overlayShowMapping: ConfigManifestEntry = {
+	id: 'OverlayShowMapping',
+	name: 'Overlay Show mapping',
+	description: 'Maps Overlay Shows to the variety of Skemas and Designs',
+	type: ConfigManifestEntryType.TABLE,
+	required: false,
+	defaultVal: [],
+	columns: [
+		{
+			id: 'Design',
+			name: 'Design',
+			rank: 0,
+			description: 'Name of the Design from the GFX Design table',
+			type: ConfigManifestEntryType.SELECT_FROM_COLUMN,
+			tableId: DESIGN_TABLE_ID,
+			columnId: DESIGN_NAME_COLUMN_ID,
+			multiple: false,
+			required: false,
+			defaultVal: ''
+		},
+		{
+			id: 'GraphicsSetup',
+			name: 'Graphics Setup',
+			rank: 1,
+			description: 'Names of the Graphics Setups',
+			type: ConfigManifestEntryType.SELECT_FROM_COLUMN,
+			tableId: GRAPHICS_SETUPS_TABLE_ID,
+			columnId: GRAPHICS_SETUPS_NAME_COLUMN_ID,
+			multiple: true,
+			required: false,
+			defaultVal: []
+		},
+		{
+			id: 'Skema',
```

Comment:
What is the reasoning for using the danish word for the id?
Would be nice if we could keep the codebase in english

There is also a mismatch between the name and the description. One uses the english word for schema, the other uses the danish. I think this should be aligned

Reply:
Good find, the value for `id` and `name` were just swapped